src/reanimated2/commonTypes.ts

```diff
@@ -23,6 +24,12 @@ export interface SharedValue<Value = unknown> {
   ) => void;
 }
 
+export interface Mutable<Value = unknown> extends SharedValue<Value> {
+  _isReanimatedSharedValue: true;
+  _animation?: AnimationObject | null; // only in Native
+  _value: Value | Descriptor | AnimatableValue;
```

Comment:
Why have you chosen this type?
Why, if `_value` is of type `Descriptor` we don't call `Mutable<Descriptor>`?

Reply:
Good idea, I changed it in 22bbdf1